__tests__/app.test.js

```diff
@@ -76,4 +77,36 @@ describe("/api/articles/:article_id",()=>{
             expect(body.msg).toBe("Bad request");
           });
       });
-})
\ No newline at end of file
+})
+
+//task 5
+describe('/api/articles', () => {
+    test('200: responds with an array of articles', () => {
+        return request(app)
+            .get('/api/articles')
+            .expect(200)
+            .then(({ body }) => {
+                body.articles.forEach(article => {
```

Comment:
if you are using a forEach to test multiple things I recommend testing for the length of the array before. If you array was empty the tests inside the forEach would be skipped but the length test would not.